var/spack/repos/builtin/packages/madgraph5amc/package.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Madgraph5amc(Package):
+    """MadGraph5_aMC@NLO is a framework that aims at providing
+       all the elements necessary for SM and BSM phenomenology,
+       such as the computations of cross sections, the generation
+       of hard events and their matching with event generators,
+       and the use of a variety of tools relevant to
+       event manipulation and analysis. """
+
+    homepage = "https://launchpad.net/mg5amcnlo"
+    url      = "https://launchpad.net/mg5amcnlo/2.0/2.7.x/+download/MG5_aMC_v2.7.3.tar.gz"
+
+    version('2.7.3.py3', sha256='400c26f9b15b07baaad9bd62091ceea785c2d3a59618fdc27cad213816bc7225')
+    version('2.7.3.py2', sha256='0b665356f4d9359e6e382e0f408dc11db594734567c6b2f0ec0e0697f2dbe099',
+            url="https://launchpad.net/mg5amcnlo/2.0/2.7.x/+download/MG5_aMC_v2.7.3.tar.gz")
+    # version('2.7.2', sha256='e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855')
+    # 2.7.2 is not available on launchpad
+
+    variant('atlas', default=False, description='Apply changes requested by ' + 
+            "the ATLAS experimenent on LHC")
+    variant('ninja', default=False, description='Use external installation' +
+            " of Ninja")
+    variant('collier', default=False, description='Use external installation' +
+            ' of Collier')
+
+    depends_on('syscalc')
+    depends_on('gosam-contrib', when='+ninja')
+    depends_on('collier', when='+collier')
+    depends_on('lhapdf')
+    depends_on('fastjet')
+    depends_on('py-six', when='@2.7.3.py3')
```

Comment:
I'm surprised that six is only needed for Python 3. Normally both Python 2 and 3 use six so that they don't have to write custom logic for each version. I have a feeling six is always needed.
```suggestion
    depends_on('py-six', type=('build', 'run'))
```

Reply:
I have checked, and `six` is not required for py2 version.